cmd/defaults.go

```diff
@@ -0,0 +1,98 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/containers/storage"
+	"k8s.io/klog/v2"
+)
+
+// Return "chroot" if we know we're not actually root, "oci" otherwise.
+func builderDefaultIsolation() (string, error) {
+	if inUserNamespace() {
+		// We probably don't have enough privileges to use a proper runtime.
+		return "chroot", nil
+	}
+	// Use the proper runtime.
+	return "oci", nil
+}
+
+// Check that /dev/fuse is usable and we have the fuse-overlayfs helper.
+func builderCanUseOverlayFUSE() string {
```

Comment:
Why isn't this returning an error?

Reply:
Sure, why not.  Changing it to return an error.